CoEpi/ui/onboarding/OnboardingViewController.swift

```diff
@@ -54,6 +55,15 @@ class OnboardingViewController: UIViewController {
         viewModel.onCloseClick()
     }
     
+    @IBAction func howDataUsedClick(_ sender: UIButton) {
+        if let url = URL(string: "https://www.hackingwithswift.com/read/") {
```

Comment:
I guess this has to be replaced?

Reply:
oops... I though I already had. 
That is fixed now.